source/proxy-dll/component/platform.cpp

```diff
@@ -79,7 +89,7 @@ namespace platform
 	public:
 		void pre_start() override
 		{
-			check_platform_registry();
```

Comment:
this change is a big no no

Reply:
my bad, it was to test something and I forgot to put it back